jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -170,15 +170,8 @@ def test_fft(self, harness: primitive_harness.Harness):
                                   "only 1D FFT is currently supported."):
         harness.dyn_fun(*harness.dyn_args_maker(self.rng()))
     else:
-      tol = None
-      if jtu.device_under_test() in ("cpu", "gpu"):
```

Comment:
It is suspicious that these tolerances are not necessary anymore. Were they only needed for some of the dtypes that are now disabled?

Reply:
No, they were generally needed. I didn't encounter any problematic instance anymore, but I guess the tests might indeed become flaky if we remove it. Adding it back.